### ISSUE TITLE
Thumbnail URLs require you to be signed in to OneDrive(?)

### ISSUE THREAD

**Issue by Mike-Bell** [microsoftgraph/microsoft-graph-docs-contrib#8322]
I'm able to get thumbnail URL's as per http://graph.microsoft.io/en-us/docs/api-reference/v1.0/api/thumbnailset_get. A logical thing I might want to do is to plop the `url` value from this response into an image tag on a HTML page. Unfortunately, this only works if the user is already signed in directly to OneDrive. If the user does not have an active login session with OneDrive (which is a legitimate possibility if I'm developing an app that uses OneDrive as backing storage), then the browser is unable to retrieve the image contents and instead returns a 403.

Is there some method for fetching pre-authenticated thumbnail URLs? I tried grabbing an access token for the OneDrive API and appending that to the URL (e.g. `preAuthUrl = url + '?access_token=' + token`), but this still 403s (I thought this method of passing an access token was supported in the OneDrive API?). I see that I can retrieve the thumbnail contents via xhr if I use my access token in the header. I guess I could try constructing a data URL from the encoded contents and putting that in my tag, but I'd like to avoid this.

TLDR wondering if there's some way to get or construct a pre-authenticated url for thumbnails. Something like an equivalent of `#content.downloadUrl / #microsoft.graph.downloadUrl` but for thumbs.
